Convex_hull_3/include/CGAL/convexity_check_3.h

```diff
@@ -187,13 +187,13 @@ bool is_strongly_convex_3(const Polyhedron& P, const Traits& traits)
 }
 
 template<class Polyhedron, class R>
-bool CGAL_is_strongly_convex_3(Polyhedron& P, Point_3<R>*)
+bool CGAL_is_strongly_convex_3(const Polyhedron& P, Point_3<R>*)
 {
   return is_strongly_convex_3(P, R());
 }
 
 template<class Polyhedron>
-bool is_strongly_convex_3(Polyhedron& P)
+bool is_strongly_convex_3(const Polyhedron& P)
 {
   typedef typename boost::property_map<Polyhedron, vertex_point_t>::type Ppmap;
```

Comment:
```suggestion
  typedef typename boost::property_map<Polyhedron, vertex_point_t>::const_type Ppmap;
```

Reply:
I changed it, but note that testsuite and examples passed locally without the change.